Source/Core/UICommon/DiscordPresence.h

```diff
@@ -4,10 +4,16 @@
 
 #pragma once
 
+#include <functional>
+
 namespace Discord
 {
+typedef std::function<void()> JoinFunction;
+
 void Init();
-void UpdateDiscordPresence();
+void InitNetPlayFunctionaly(const JoinFunction&);
+void CallPendingCallbacks();
+void UpdateDiscordPresence(const int party_size = 0, const char* host = "");
```

Comment:
`const` on `int` in a prototype can be removed, as it's ignored at a type system level. It can be left on the definition, however. It's also likely not a great idea to use `const char*` for a string that cannot be null.